deploy/rhos-dashboard.yaml

```diff
@@ -2394,156 +1540,165 @@ spec:
-          "panels": [],
-          "title": "Disk/File System",
-          "type": "row"
-        },
-        {
+          "aliasColors": {},
+          "bars": false,
           "cacheTimeout": null,
+          "dashLength": 10,
+          "dashes": false,
           "datasource": "STFPrometheus",
-          "description": "",
+          "fill": 1,
+          "fillGradient": 0,
           "gridPos": {
-            "h": 6,
-            "w": 3,
-            "x": 0,
-            "y": 92
+            "h": 5,
+            "w": 12,
+            "x": 12,
+            "y": 29
+          },
+          "hiddenSeries": false,
+          "id": 23,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
           },
-          "id": 50,
+          "lines": true,
+          "linewidth": 1,
           "links": [],
+          "nullPointMode": "null",
           "options": {
-            "fieldOptions": {
-              "calcs": [
-                "lastNotNull"
-              ],
-              "defaults": {
-                "decimals": 1,
-                "mappings": [
-                  {
-                    "id": 0,
-                    "op": "=",
-                    "text": "N/A",
-                    "type": 1,
-                    "value": "null"
-                  }
-                ],
-                "max": 1,
-                "min": 0,
-                "nullValueMode": "connected",
-                "thresholds": [
-                  {
-                    "color": "#299c46",
-                    "value": null
-                  },
-                  {
-                    "color": "rgba(237, 129, 40, 0.89)",
-                    "value": 0.9
-                  },
-                  {
-                    "color": "#d44a3a",
-                    "value": 0.95
-                  }
-                ],
-                "unit": "percentunit"
-              },
-              "override": {},
-              "values": false
-            },
-            "orientation": "horizontal",
-            "showThresholdLabels": false,
-            "showThresholdMarkers": true
+            "dataLinks": []
           },
-          "pluginVersion": "6.5.1",
+          "percentage": false,
+          "pluginVersion": "6.5.2",
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(collectd_df_df_complex{plugin_instance!~\"devtmpfs|overlay|shm|tmpfs\",type_instance!=\"free\",host=\"$hosts\"})/sum(collectd_df_df_complex{plugin_instance!~\"devtmpfs|overlay|shm|tmpfs\",host=\"$hosts\"})",
+              "expr": "sum(collectd_hugepages_vmpage_number{type_instance=\"free\",host=\"$hosts\"})",
```

Comment:
This appears to have changed from "type_instance=used" to "type_instace=free", which is not made clear in the title, legend, or labels. To be useful I think the graph needs to represent both the number being used, and what % that is of the total hugepages, if possible.

Reply:
Im amazed you noticed that just by looking through the json data. That should never have been changed. For the update, I am including a separate guage panel that will show percentage of hugepages used.